org.koreader.KOReader.yml

```diff
@@ -0,0 +1,58 @@
+app-id: org.koreader.KOReader
```

Comment:
`koreader.org` doesn't exist. Also no mention of that domain in other reference. Is this app-id valid?

Reply:
it is `koreader.rocks`

Reply:
then why did you use `koreader.org` ?

Reply:
I don't know :p

I came from https://github.com/koreader/koreader/issues/6349#issuecomment-1356721925 and have no relation with the person who's PR'ing this :)

Reply:
oh sorry I thought you were the submitter.

Apologies.

Anyway. app-id as submitted doesn't meet the requirements.

Reply:
so why is there even an appdata file here then?

Reply:
To clarify further, it is a requirement for the domain to exist, as it is tied to upcoming verification and payments/donations features.